app/models/customer.py

```diff
@@ -0,0 +1,27 @@
+from flask import current_app
+from app import db
+from datetime import datetime
+from sqlalchemy.orm import backref, relationship
+
+class Customer(db.Model):
+    customer_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    registered_at = db.Column(db.DateTime, nullable = True)
+    videos_checked_out_count = db.Column(db.Integer, default = 0) 
+
+    # One to many relationship: one customer has many rentals 
+    customers = db.relationship("Rental", backref="customers", lazy=True)
```

Comment:
It looks like this should actually be `rentals`, but you've already dealt with it on the Rental model, so I don't believe you need this here.